src/main/java/com/google/codeu/servlets/MessageFeedServlet.java

```diff
@@ -27,40 +27,40 @@
 @WebServlet("/feed")
 public class MessageFeedServlet extends HttpServlet{
 
- private Datastore datastore;
```

Comment:
Probably want to fix the indentation, here and everywhere, to be consistent with the rest of the file. Check if you IDE has the indentation settings for tabs/ space set appropriately

Reply:
This was a problem in the original commit, but has been fixed. It should be fine now!